core/platform/models_test.py

```diff
@@ -361,6 +367,25 @@ def test_import_search_services(self):
             self.registry_instance.import_search_services(),
             elastic_search_services)
 
+    def test_import_storage_services(self):
+        """Tests import taskqueue services function."""
+
+        class MockCloudStorage():
+            pass
+
+        with self.swap(constants, 'EMULATOR_MODE', False):
+            # Mock Cloud Storage since importing it fails in emulator env.
+            sys.modules['core.platform.storage.cloud_storage_services'] = (
+                MockCloudStorage
+            )
+            self.assertEqual(
+                self.registry_instance.import_storage_services(),
+                MockCloudStorage)
+        from core.platform.storage import dev_mode_storage_services
```

Comment:
Maybe add a line above this one since it's unrelated to what comes above it (right?)

Reply:
Right. Done.